[INFO] | 2024-11-10 07:41:54 AM | analyzer.0x2aeee16d8412beabc5f38a6995ee5e4d32ab7f9d | Analysis started at: 2024-11-10 07:41:54 AM
[INFO] | 2024-11-10 07:41:56 AM | analyzer.0x2aeee16d8412beabc5f38a6995ee5e4d32ab7f9d | Version constraints: ['^0.8.17']
[INFO] | 2024-11-10 07:41:56 AM | analyzer.0x2aeee16d8412beabc5f38a6995ee5e4d32ab7f9d | solc version: v0.8.17
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/crytic_compile/platform/solc.py", line 368, in _run_solc
    ret = json.loads(stdout)
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.8-py3.6.egg/slither/slither.py", line 60, in __init__
    crytic_compile = CryticCompile(target, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/crytic_compile/crytic_compile.py", line 137, in __init__
    self._compile(**kwargs)
  File "/usr/local/lib/python3.6/dist-packages/crytic_compile/crytic_compile.py", line 987, in _compile
    self._platform.compile(self, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/crytic_compile/platform/solc.py", line 152, in compile
    force_legacy_json=force_legacy_json,
  File "/usr/local/lib/python3.6/dist-packages/crytic_compile/platform/solc.py", line 372, in _run_solc
    raise InvalidCompilation(f"Invalid solc compilation {stderr}")
crytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Invalid option to --combined-json: compact-format


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "contractlint.py", line 71, in analyze_contracts
    slither_obj = Slither(contract_path, solc=solc_path)
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.8-py3.6.egg/slither/slither.py", line 63, in __init__
    raise SlitherError('Invalid compilation: \n'+str(e))
slither.exceptions.SlitherError: Invalid compilation: 
Invalid solc compilation Invalid option to --combined-json: compact-format


real	0m3.839s
user	0m5.773s
sys	0m3.268s
